x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['common', 'timePicker', 'hosts', 'settings']);
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+
+  describe('Endpoint Alert Resolver', function () {
+    this.tags(['ciGroup7']);
+
+    before(async () => {
+      await esArchiver.load('endpoint/resolver_tree/api_feature', { useCreate: true });
```

Comment:
I'm concerned about the size of this archive. It is large and not compressed. Looking at the nature of the tests, I think an archive that has a single event might be enough.

Reply:
It will not be enough for later tests we will add more for different events

Reply:
We also will be adding archive for MAC and Linux

Reply:
The tests in this PR use the on-screen map controls to pan the map. The archive is a 25.8MB json file. It is nearly a million lines long and it is not compressed. Here are my concerns:

* It isn't clear what the archive contains or why it is needed to pan the map
* If it contains fixtures for tests that you plan to add in the future then I'd like to know what those tests are and why they require a 25MB fixture. I don't think we should merge unused code (including fixtures.) 
* a 25MB json file is too large. If that much data is needed to test Resolver then I think we should discuss having several individual archives (each compressed.)

Please either remove the unused data or let's discuss it.

Reply:
Igor, we can move to using more targeted archives for each test or set of tests and only keeping the events we're analyzing in the data.  We can either load the archived data and delete the extraneous events through elastic before rearchiving or we can unzip the data from esarchiver and directly modify the json and rearchive.  I can try walking through that with you.

Reply:
Ok We can do that